hello-microverse.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```

Comment:
- [ ] Please from the project requirement, your CSS file should include one class with styles for your header (make it in your favorite color). 
- [ ] Also, I can see this file is not linked to your HTML file.

Reply:
Hello Desmond, I think now it's done, please let me know if I have to make more changes, Thanks a lot.